src/models/differentiable/other/laplace_redux.jl

```diff
@@ -31,5 +31,6 @@ function LaplaceReduxModel(model; likelihood::Symbol=:classification_binary)
 end
 
 # Methods
-logits(M::LaplaceReduxModel, X::AbstractArray) = M.model.model(X)
-probs(M::LaplaceReduxModel, X::AbstractArray) = LaplaceRedux.predict(M.model, X)
+# The methods are commented out for the time being, as they are not called from anywhere in the package
+# logits(M::LaplaceReduxModel, X::AbstractArray) = M.model.model(X)
+# probs(M::LaplaceReduxModel, X::AbstractArray) = LaplaceRedux.predict(M.model, X)
```

Comment:
I don't see why this is a reason to comment them out

Reply:
Otherwise they would be included in the test coverage report, which does not really make sense for methods which have not been properly implemented yet and are thus untestable, especially since they're not called from anywhere